scss/components/_badge.scss

```diff
@@ -18,6 +18,10 @@ $badge-color: $white !default;
 /// @type Color
 $badge-color-alt: $black !default;
 
+/// Coloring classes. The names in this list output classes in your CSS, like `.secondary`, `.success`, and so on. Each value must also be in the `$foundation-palette` map.
+/// @type List
+$badge-classes: ( secondary success warning alert ) !default;
```

Comment:
But should the default be…
```
$badge-palette: map-remove( $foundation-palette, (primary) ) !default;
```
…so that we don't generate an unnecessary `.primary` class? That's one thing I was trying to do with the `$badge-palette` list.